src/main/resources/application.properties

```diff
@@ -166,3 +166,6 @@ ccd.am.read.from_am=${CCD_AM_READ_FROM_AM:}
 # http statuses with comma separated to ignore audit logs
 audit.log.ignore.statues=404
 audit.log.enabled=true
+
+# List of roles which are comma separated
+ccd.role.whitelist=caseworker-caa
```

Comment:
```suggestion
ccd.access-control.cross-jurisdictional-roles=caseworker-caa
```

Reply:
furthermore, if we introduced an env var with its value set in `values.yaml`, that would allow us add other cross-jurisdictional-roles in the future just by changing the flux config, without having to do any DataStore release.

Reply:
👍  done